server/src/api/tag.ts

```diff
@@ -12,6 +13,28 @@ router.get('/', authenticateToken, async (req: AuthorizedRequest, res) => {
   res.json(tags);
 });
 
+router.get('/:user_id', authenticateToken, async (req: AuthorizedRequest, res) => {
+  const { userIdx } = req.user;
+  const { user_id: friendId } = req.params;
+
+  try {
+    const friend = (await executeSql('select idx from user where user_id = ?', [friendId])) as RowDataPacket;
```

Comment:
구조분해할당을 적용하면 조금 더 읽기 좋아질 것 같습니다.
(가벼운 코멘트입니다 지금도 충분히 이해가 잘 됩니다)
```suggestion
    const [friend] = (await executeSql('select idx from user where user_id = ?', [friendId])) as RowDataPacket;
```

Reply:
우선 제안해주신 방식을 사용하려면 아래와 같이 사용해야 합니다!
```ts
const [friend] = (await executeSql('select idx from user where user_id = ?', [friendId])) as RowDataPacket[];
```
또 이런 방식을 사용하여 해당 친구가 없을 경우 `friend === undefined`인 경우로 처리할 수도 있긴 하지만 어차피 `idx`는 또 구조 분해 할당을 통해 꺼내서 사용해야해서 그냥 해당 PR과 같이 코드를 작성하였습니다!
제안 감사합니다~~!